src/utils/src/deviceinfo.cpp

```diff
@@ -16,49 +16,49 @@
  */
 
 #include "deviceinfo.h"
+#include <QFile>
+#include <QCoreApplication>
+
+const char* CONFIG_FILE = "/etc/machine.conf";
```

Comment:
Let's make a subdirectory for qml-asteroid. Or at least for asteroid. I don't like using the /etc namespace for something as generic as "machine.conf"

Reply:
Moved to a subdirectory named `qml-asteroid`.